### ISSUE TITLE
Amazon.nl

### ISSUE THREAD

**Issue by Atydin** [philipmulcahy/azad#98]
Hi,

Thank you for your amazing work. can you possibly add amazon.nl to your code?

Thank you

**Comment by philipmulcahy** [comment]
You're welcome Atydin.

Please send me two dump files a) for a physical order and  b) for a digital order and I'll get to it when I have time. (The dump file saves amazon's html for the orders and makes it possible for me to see the pages .nl would show to me if I had your account details, but only for the orders you send me).
Just to be as safe as possible send them to the new and shiny azadextension@gmail.com, including a link to this ticket, and then ping me here (not checking that account too often unless prompted).
DON'T attach them here unless you want everyone on the internet to be able to see them. I'm not aware of any way "they" could own your account from the dump, but why take the risk?

Yours,

Philip

**Comment by xavbart** [comment]
Same process to add amazon.fr ?

**Comment by philipmulcahy** [comment]
Hi @xavbart I think it works on .fr already (though having some test data that isn't mine would be great). Let's start a new issue and not mix it up with .nl

**Comment by xavbart** [comment]
Ok stupid me. I does work indeed. I just followed Rob's example (he recommended you here https://robservatory.com/download-and-track-amazon-purchases-over-time/ ) and I did not see what he showed in screenshot because my eyes just saw the [IDLE] button, and NOT THE WHOLE LIST OF YEARS. Apologies and I may delete my responses here if you want me to hide in shame ;-) I'll test and let you know if any issue arises.
